apps/meteor/app/push/server/fcm.ts

```diff
@@ -145,12 +152,7 @@ function getFCMMessagesFromPushData(userTokens: string[], notification: PendingP
 	return userTokens.map((token) => ({ message: { ...message, token } }));
 }
 
-export const sendFCM = function ({ userTokens, notification, _replaceToken, _removeToken, options }: NativeNotificationParameters): void {
-	// We don't use these parameters, but we need to keep them to keep the function signature
-	// TODO: Remove them when we remove the old sendGCM function
-	_replaceToken;
-	_removeToken;
-
```

Comment:
I think we could, not sure if it is worth it though.
Since these parameters are still being passed to the `sendFCM` I am just not destructuring the `_replaceTokens` one, since we don't need it here.
I was being dumb in this PR, instead of adding this comment I could've just not desctructured them :)